internal/cli/renderer/renderer.go

```diff
@@ -24,7 +24,7 @@ func RenderSimulation(s *apimodel.Simulation) (string, error) {
 	buf := strings.Builder{}
 	renderHeader := func(cmd apimodel.Command) string { return "Command will" }
 
-	command, err := renderCommand(s.Command, renderHeader, formatSimulatedResourceUpdate)
+	command, err := renderCommand(s.Command, renderHeader, formatSimulatedResourceUpdate, true)
```

Comment:
Can we use the same pattern as for the resource update and pass in a func that renders the thing? That way we don't have boolean flags as parameters and if-statements. The intent of the previous refactoring was to keep the renderCommand method not simulation-aware.